<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QWidget" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>60</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="searchBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>29</y>
     <width>32</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/search.png</normaloff>../icons/search.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResults">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
